### DISCUSSION TITLE
Windows 10 Search Bar Explorer Patcher

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2545] [Ideas]
Hello, I like Explorer Patcher and everything else that it features to make the system look like Windows 10. The only thing that does not satisfy me is the search bar. It is just a tiny button. Is it possible for the developers to make it bigger/a rectangle in a future update?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by TimMarvelHero** [reply]
What is that needed for?

**Reply by pyrates999** [reply]
1. In case your windows version is out of date
2. In case your EP version is out of date
3. In case a setting in EP is causing this

**Reply by TimMarvelHero** [reply]
It is not a glitch that's doing that, the search button is there, I want the search button to be, I don't know, be longer I suppose?

**Reply by TimMarvelHero** [reply]
I don't think making the search bar longer is available on Explorer Patcher, that's why I want it to possibly exist in the future

**Reply by TimMarvelHero** [reply]
and plus, I have Windows 11, and also on the latest version.

**Reply by pyrates999** [reply]
Please provide the details I'm asking for.

**Reply by TimMarvelHero** [reply]
The 3 reasons are not caused by this, there is no settings to change this look.

**Reply by TimMarvelHero** [reply]
![C0B39986-D548-470E-B410-B0F4851C6C02](https://github.com/valinet/ExplorerPatcher/assets/152800265/1de03a35-e2e7-4a16-90ff-e87a7fd2aca5)

**Reply by milestprower92** [reply]
No longer possible in 22H2 and 23H2 because Microsoft has removed the code of the old search box from the old taskbar and there are no plans to have it added back due to the fact that the search icon does the same thing as the search box.
